core/chains/evm/config/toml/config.go

```diff
@@ -759,19 +759,27 @@ func (u *FeeHistoryEstimator) setFrom(f *FeeHistoryEstimator) {
 }
 
 type DAOracle struct {
-	OracleType             OracleType
+	OracleType             DAOracleType
 	OracleAddress          *types.EIP55Address
 	CustomGasPriceCalldata string
 }
 
-type OracleType string
+type DAOracleType string
```

Comment:
cc @jmank88 I also updated this to `DAOracleType` to be more descriptive since it will continue to live in the `toml` package, and we won't be violating: https://go.dev/doc/effective_go#names